conda_gitenv/deploy.py

```diff
@@ -54,6 +54,11 @@ def deploy_tag(repo, tag_name, target):
 
 
 def create_env(pkgs, target, pkg_cache):
+    # Empty package cache so that it will reinitialised.
+    conda.install.package_cache_ = {}
```

Comment:
Also isn't there a conda api function to control the package cache? rather than stomping over the `package_cache_`?